app/helpers/reports_helper.rb

```diff
@@ -10,4 +10,21 @@ def report_options
       "Other"
     ]
   end
+
+  def report_model_property(report, model, property)
+    if report_has_model_property?(report, model, property)
+      return report.properties[model][property]
+    else
+      return ""
+    end
+  end
+
+  private
+  def report_has_model_property?(report, model, property)
+    return false unless report.respond_to? :properties
```

Comment:
I've never seen `.respond_to?` before, but won't this always be true?

Reply:
[Documentation on respond_to](https://apidock.com/rails/ActionController/MimeResponds/InstanceMethods/respond_to)

This check is to handle old reports whose properties field will be `nil` since it was created without any properties. Nil does not respond to the properties method call, and thus the check will return early.

Reply:
If the `respond_to?` check wasn't there, the method would error when trying to process old reports.